_episodes/01-tidiness.md

```diff
@@ -3,79 +3,101 @@ title: "Data Tidiness"
+other later. Using dates also helps create unique identifiers, because even
+if you process the same sample twice, you don't usually do it on the same
+day, or if you do, you're aware of it and give them names like A and B.
 
-# Exercises
+> #### Unique identifiers
+> Unique identifiers are a unique name for a sample or set of sequencing data.
+> They are names for that data that only exist for that data. Having these
+> unique names makes them much easier to track later.
+{: .callout}
 
-## **A. Sending samples to the facility**
+## Data about the experiment
 
-The first step in sending your sample for sequencing will be to complete a form documenting the metadata for the facility. Take a look at the following submission spreadsheet. 
+Data about the experiment is usually collected in spreadsheets, like Excel.
 
-[Sample submission sheet](./sample_submission.txt)*<br> **FIXLINK**
-> *Download the file using right-click (PC)/command-click (Mac). This is a tab-delimited text file; try opening it with Excel or another spreadsheet program. 
+What type of data to collect depends on your experiment and there are often guidelines from metadata standards.
 
-### **Questions**
+> #### Metadata standards
+> Many fields have particular ways that they structure their metadata so it's
+consistent and can be used across the field.
+>
+> The Digital Curation Center maintains [a list of metadata  standards](http://www.dcc.ac.uk/resources/metadata-standards/list). Ones particularly relevant for genomics data are from the [Genomics Standards Consortium](http://gensc.org/projects/).
+>
+> If there aren't metadata standards already, you can think about what the minimum amount of information someone would need to know about your data to be able to work with it, without talking to you.
+>
+{: .callout}
 
-1. What are some errors you can spot in the data?
--  Typos, missing data, inconstancies?
-2. What improvements could be made to the choices in naming?
-3. What are some errors in the spreadsheet that would be difficult to spot? Is there anyway you can test this?
+### Structuring data in spreadsheets
 
+Independent of the what type of data you're collecting there are standard ways to enter that data into the spreadsheet, to make it easier to analyze later. We often enter data that makes it easy for us as humans to read and work with it, because we're human! Computers need data structured in a way that they can use it, so to use this data in a computational workflow, we need to think like computers when we use spreadsheets.
 
-## **B. Retrieving samples from the facility**
+The cardinal rules of using spreadsheet programs for data:
 
-When the data come back from the facility, you will receive some documentation (metadata) as well as the sequence files themselves. Download and examine the following file - here provided both as an excel and a text file:
+- Put all your variables in columns - the thing you’re measuring, like ‘weight’ or ‘temperature’.
```

Comment:
change 'weight' and 'temp' to variables that would be included in genomic metadata table. E.g. 'strain name' and 'library prep method'